src/app/features/reporting/reporting/reporting.component.spec.ts

```diff
@@ -328,4 +328,23 @@ describe("ReportingComponent", () => {
         expect(component.errorDetails).not.toBeNull();
       });
   });
+
+  it("should return raw data for version 1 SQL reports", () => {
+    const mockData = [
+      { key: "Total students", value: 6 },
+      { key: "Students gender", value: 10 },
```

Comment:
isn't this the format of v2 reports?

I would like to have two tests actually - one for v1 data and one for v2 data